example/lib/page_crash_reporting.dart

```diff
@@ -73,6 +67,13 @@ class CrashReportingPage extends StatelessWidget {
             MyButton(text: 'Throw Native Exception', color: 'orange', onPressed: throwNativeException),
             MyButton(text: 'Record Exception Manually', color: 'teal', onPressed: recordExceptionManually),
             MyButton(text: 'Divided By Zero Exception', color: 'teal', onPressed: dividedByZero),
+            MyButton(
```

Comment:
no new buttons

Reply:
This new button is to specifically test that async errors is caught by `PlatformDispatcher.instance.onError`. Formally, it was caught by runZoned, but now we've removed that

Reply:
Should I remove it?

Reply:
no I mean there are no new buttons to test these errors without a try catch block

Reply:
I told you to add a button for each unique crash